gpslogger/build.gradle

```diff
@@ -188,10 +188,14 @@ dependencies {
     //Markdown to HTML library
     implementation 'com.github.commonsguy:cwac-anddown:dedb48ec0f'
 
-//    implementation 'com.github.mwiede:jsch:0.1.58'
-    implementation group: 'com.jcraft', name: 'jsch', version: '0.1.54'
-    implementation group: 'com.jcraft', name: 'jzlib', version: '1.1.3'
+    implementation 'com.github.mwiede:jsch:0.1.67'
+    implementation 'org.bouncycastle:bcpkix-jdk15to18:1.68'
+    implementation 'org.bouncycastle:bcprov-jdk15to18:1.68'
 
+    //File Picker, but a patched version
+    //https://github.com/codekidX/storage-chooser/issues/127
+    //https://github.com/codekidX/storage-chooser/issues/134
+    implementation 'com.github.mendhak:storage-chooser:2.0.4.4b'
```

Comment:
My fork of the storage-chooser, with [some fixes](https://github.com/mendhak/storage-chooser/releases/tag/2.0.4.4b)